scripts/pi-hole/js/queries.js

```diff
@@ -200,6 +200,12 @@ function parseQueryStatus(data) {
       buttontext =
         '<button type="button" class="btn btn-default btn-sm text-red btn-blacklist"><i class="fa fa-ban"></i> Deny</button>';
       break;
+    case "SPECIAL_DOMAIN":
+      colorClass = "text-red";
+      icon = "fa-solid fa-ban";
+      fieldtext = data.status;
+      buttontext = "";
+      break;
```

Comment:
The status is showing as expected (`SPECIAL_DOMAIN`) and also a red "ban" icon, but I think we should also mark the row as blocked (red background - to match the icon).

```suggestion
      buttontext = "";
      blocked = true;
      break;
```

![PR2895](https://github.com/pi-hole/web/assets/1385443/63d49efa-8763-4494-9a45-90ce320b57c0)